yolo.js

```diff
@@ -0,0 +1,92 @@
+var model = [
+    {
+        name: "cat 1",
+        imageUrl: "https://images.pexels.com/photos/104827/cat-pet-animal-domestic-104827.jpeg?auto=compress&cs=tinysrgb&h=350",
+        clickCount: 0,
+        id: 0
+    },
+    {
+        name: "cat 2",
+        imageUrl: "https://images.pexels.com/photos/20787/pexels-photo.jpg?auto=compress&cs=tinysrgb&h=350",
+        clickCount: 0,
+        id: 1
+    }
+];
+
+var controller = {
+    init: function(){
+        view.renderList(model)
+        view.renderCat(0)
+    },
+    countAddOne: function(catId){
+        model[catId]["clickCount"] ++
+        view.updateCount(catId)
+    },
+    updateCat: function(catId){
+        model[catId]["name"] = $("#newName").val()
+        model[catId]["imageUrl"] = $("#newImageUrl").val()
+        model[catId]["clickCount"] = $("#newNumberOfClicks").val()
+        
+        view.closeAdmin()
+        view.renderCat(catId)
+    }
+}
+
+var view = {
+    renderList: function(model){
+        for (cat in model){
+            $(".collection").append('<a href="#!" class="collection-item" onClick="view.renderCat(' + model[cat]["id"] + ')">' + model[cat]["name"] + '</a>')
+        }
+    },
+    renderCat: function(catId){
+
+        if ($("#catContent").children().length != 0) {
```

Comment:
Uporabi striktno preverjanje (trojni enacaj), ce ves da imas enake type (Number v tem primeru). Je hitrejsi in korektnejsi.
` if ($("#catContent").children().length !== 0)`